linters/terrascan/plugin.yaml

```diff
@@ -16,15 +16,25 @@ lint:
     - name: terrascan
       download: terrascan
       known_good_version: 1.18.1
-      files: [terraform]
       suggest_if: config_present
       commands:
         - name: lint
           output: sarif
           is_security: true
           run: terrascan scan --iac-file ${target} --output sarif
+          read_output_from: stdout
+          files: [terraform] # terrascan supports - arm, cft, docker, helm, k8s, kustomize, terraform. However, we only support terraform and docker as filetypes.
```

Comment:
does this run command not need a `-i terraform` or anything?

Reply:
that's the default, but specified terraform to be more explicit.